invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessUsersGroups/UserGroupAccessSearchResult.js

```diff
@@ -0,0 +1,313 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2021-2024 CERN.
+// Copyright (C) 2021 Northwestern University.
+// Copyright (C) 2021 Graz University of Technology.
+// Copyright (C) 2023 TU Wien.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+import React, { Component } from "react";
+import { Table, Button, Item, Label, Grid } from "semantic-ui-react";
+import { withCancel, http } from "react-invenio-forms";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import PropTypes from "prop-types";
+import { SuccessIcon } from "@js/invenio_communities/members";
+import { Image, dropdownOptionsGenerator, ErrorMessage } from "react-invenio-forms";
+import { AccessDropdown } from "../AccessLinks/AccessDropdown";
+import { dropdownOptions } from "../AccessLinks/LinksSearchResultContainer";
+import { AddUserGroupAccessModal } from "./AddUserGroupAccessModal";
+
+const accessDropdownOptions = [
+  ...dropdownOptions,
+  {
+    key: "manage",
+    name: "manage",
+    text: i18next.t("Can manage"),
+    title: i18next.t("Can manage"),
+    value: "manage",
+    description: i18next.t(
+      "Can manage access, edit drafts and view restricted records/files."
+    ),
+  },
+];
+
+class UserGroupAccessSearchResultItem extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { loading: false, success: undefined, error: undefined };
+  }
+  handleDelete = async () => {
+    const { result, endpoint } = this.props;
+    this.setState({ loading: true });
+    const cancellableAction = withCancel(
+      http.delete(`${endpoint}/${result?.expanded?.subject?.id}`)
+    );
+    try {
+      await cancellableAction.promise;
+      this.setState({ loading: false, success: true });
+      this.handleSuccess();
+    } catch (error) {
+      this.setState({ loading: false, error: error });
+      console.error(error);
+    }
+  };
+  handleSuccess = () => {
+    const { addedOrDeletedCallback, callbackEndpoint, searchType } = this.props;
+    addedOrDeletedCallback(callbackEndpoint, searchType);
+  };
+
+  render() {
+    const { result, permissions, endpoint, searchType, updatedCallback } = this.props;
+    const { loading, success, error } = this.state;
+    const entity = result.expanded.subject;
+    return (
+      <Table.Row>
+        {error ? (
+          <ErrorMessage
```

Comment:
same comment as somewhere before: do we need the error message to be displayed inside the table?